projects/extension/sql/idempotent/001-openai.sql

```diff
@@ -154,7 +154,7 @@ create or replace function ai.openai_chat_complete
 , temperature float8 default null
 , top_p float8 default null
 , tools jsonb default null
-, tool_choice jsonb default null
+, tool_choice text default null
```

Comment:
I guess there is no unions in sql functions? :D

Reply:
Unfortunately no